docs/visual/e2e-testing/setup.md

```diff
@@ -495,6 +495,11 @@ static void test() {
 Now you can run your test and [view your results](#view-test-results) in the Visual Testing Dashboard. Be sure to [review and accept](#accept-baseline) your baseline.
 
 
+## Setting Timeouts
+
+If you're using several visual assertions (such as [snapshots](/visual/e2e-testing/commands-options/#snapshot-command)) in your WebdriverIO test, you may need to increase the `timeout` value in your configuration to help with test stability.
```

Comment:
Thanks for this PR @kimsauce , can we link to this url from WDIO regarding the timeouts, that makes it easier for customers to adjust and we don't need to provide an example https://webdriver.io/docs/timeouts/#framework-related-timeouts

I'm also not sure how this works in other language, if they also use step / method timeouts, maybe we can also make a generic remark about that because now it's only WDIO focussed

Reply:
Hi @wswebcreation , thank you for your feedback! I've changed up this section a bit. Let me know what you think: https://github.com/saucelabs/sauce-docs/pull/959/files.